lib/processTextStyles.js

```diff
@@ -40,12 +41,24 @@ async function processTextStyles(textStyles, outputFolder) {
         addOptionalProperty("blur", data.style, textBlur(allStyles))
         addOptionalProperty("underline", data.text, textUnderline(encodedAttributes))
         addOptionalProperty("strikethrough", data.text, textStrikethrough(encodedAttributes))
-        ret.push(data)
+        styles.push(data)
+
+        // Get properties
+        const propertyData = {
+          name: style.name,
+          properties: [
+            `.textColor(UIColor(red: ${color.red}, green: ${color.green}, blue: ${color.blue}, alpha: ${color.alpha}))`,
+            `.kerning(${encodedAttributes.kerning})`,
+            `.font(UIFont(name: "${fontAttributes.name}", size: ${fontAttributes.size})!)` // using ! to unwrap the font. Should we do this? IMHO the font not being there is a programmer error
```

Comment:
This should follow the name / value pattern of the SwiftUI for consistency, unless there’s a good reason not to :-)